src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java

```diff
@@ -326,6 +329,14 @@ protected void addAdditionalContextValues(final String action, final TransportRe
             }
         }
 
+        String extensionUniqueId = getThreadContext().getHeader("extension_unique_id");
```

Comment:
For follow up PR, can we put this in a constant in core and use that constant here?